packages/management/pinniped-config-controller-manager/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# TODO: Makefile to hook into root Makefile?
```

Comment:
Also, should we add an `OWNERS` path for ourselves in this PR so we can continue to merge stuff w/o external approval?